h/services/subscription.py

```diff
@@ -0,0 +1,96 @@
+from typing import List
+
+from sqlalchemy import func
+from sqlalchemy.orm import Session
+from webob.cookies import SignedSerializer
+
+from h.models import Subscriptions
+from h.security import derive_key
+
+
+class InvalidUnsubscribeToken(Exception):
+    """When an unsubscribe token has an invalid format."""
+
+
+class SubscriptionService:
+    """A service for managing user communication preferences."""
+
+    MISSING_SUBSCRIPTION_ACTIVE = True
+    """Default behavior when we find a missing subscription."""
```

Comment:
This is never going to get changed, but I wanted to shout out about the fact a choice is being made here.